infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/KubernetesDeployments.java

```diff
@@ -527,8 +526,22 @@ public void eventReceived(Action action, Event event) {
               if (POD_OBJECT_KIND.equals(involvedObject.getKind())
                   || REPLICASET_OBJECT_KIND.equals(involvedObject.getKind())
                   || DEPLOYMENT_OBJECT_KIND.equals(involvedObject.getKind())) {
-
+                LOG.debug("Event {}", event);
```

Comment:
Is it really something we would like to introduce on debug level?.. Previously debug logs were quite minimal but useful, with it I'm not sure they will keep being useful unless you filter them

Reply:
+1, in my opinion this should be trace log

Reply:
removed.